cluster-autoscaler/README.md

```diff
@@ -87,12 +87,11 @@ target ETA and the actual releases.
 
 | Date       | Maintainer Preparing Release | Backup Maintainer |
 |------------|------------------------------|-------------------|
-| 2023-03-15 | MaciekPytel                  | gjtempleton       |
-| 2023-05-17 | gjtempleton                  | BigDarkClown      |
-| 2023-07-19 | BigDarkClown                 | towca             |
-| 2023-09-13 | towca                        | x13n              |
-| 2023-11-15 | x13n                         | MaciekPytel       |
 | 2024-01-17 | MaciekPytel                  | gjtempleton       |
+| 2024-03-13 | gjtempleton                  | BigDarkClown      |
+| 2024-05-22 | BigDarkClown                 | towca             |
```

Comment:
One minor comment - would you mind swapping backup shifts @towca? I would take this one and you'd take the one in September. 
My 10th marriage anniversary is in September and I estimate a very high chance that I will take some vacation to travel at this time :)

Reply:
After including minor releases this is no longer an issue, your anniversary travel plans have been de-risked now :)